doc/tutorials/modeling-threat-intel-ctim.md

```diff
@@ -0,0 +1,702 @@
+
+#### 1.5.1: Sighting Field Summary
+
+##### 1.5.1.1: Required Sighting Fields
+
+- **observed_time**:
+- **confidence**: Must be one of `["Info", "Low", "Medium", "High", "None", "Unknown"]`
+- **count**: The number of times the observable was seen
+
+##### 1.5.1.2: Optional Sighting Fields
+
+- **observables**: The object(s) of interest, structured as an observable, defined above.
+- **relations**: The are relations within a sighting which help provide any context we can about where the observable(s) came from.  See below for more information.
+- **internal**: A boolean value describing if this sighting is internal to our network.
+- **severity**: Must be one of `["Info", "Low", "Medium", "High", "None", "Unknown"]`
+- **resolution**: Must be one of `["detected" "blocked" "allowed" "contained"]`
+- **sensor**: The OpenC2 Actuator name that best fits the device that is creating this sighting (e.g. network-firewall, sensor, endpoint, network-device, human).  Valid sensor types are enumerated in the [CTIM vocabularies schema](https://github.com/threatgrid/ctim/blob/74857ac6ffed206b3dcf01f171feb30e08277191/src/ctim/schemas/vocabularies.cljc#L334).
+- **targets**: An enumeration of target devices where the sighting came from. See below for more information.
+
+#### 1.5.2: Targets
+
+**Target** entries are structured as follows:
+
+##### 1.5.2.1: Target Field Summary
+
+###### 1.5.2.1.1: Required Target Fields
+
+- **type**: Must be a sensor type, as defined above.
+- **observables**: Must be a vector of observables, as defined above.
+- **observed_time**: The time at which the observable was seen.
+
+###### 1.5.2.1.2: Optional Target Fields
+
+- **os**: Operating system name
+- **properties_data_tables**: A URI leading to a data table.
+
+##### 1.5.2.2: Example Target
+
+```json
+{"type": "network.firewall",
+ "observables": [{"type": "ip", "value": "187.75.16.75"}],
+ "observed_time": {"start_time" : "2019-03-01T20:01:27.368Z"}}
+```
+
+#### 1.5.3: Observables vs Observed Relations
+
+Earlier we discussed that not everything we are *able* to observe merits being turned into an observable.  For sightings, this is made even more explicit by the inclusion of the `relations` field, which allows us to provide additional context about the observable that is the object of the sighting.
+
+For example, imagine if we have a known malicious domain `baddomain.com`.  At the time that we saw the domain being contact (triggering the creation of a Sighting), we might know that the domain resolved to the IPv4 address `8.8.8.8`.  We know from months of tracking this malicious domain that it is bad news.  However, we would **not** create a 2nd Judgement on the associated IPv4 address.  This is because the IP isn't the actual observable that triggered our malicious judgement.  It's useful context, but it isn't actually a malicious IP address.  (It's actually the IP address of Google's DNS servers).
```

Comment:
```suggestion
For example, imagine if we have a known malicious domain `baddomain.com`.  At the time that we saw the domain being contacted (triggering the creation of a Sighting), we might know that the domain resolved to the IPv4 address `8.8.8.8`.  We know from months of tracking this malicious domain that it is bad news.  However, we would **not** create a 2nd Judgement on the associated IPv4 address.  This is because the IP isn't the actual observable that triggered our malicious judgement.  It's useful context, but it isn't actually a malicious IP address.  (It's actually the IP address of Google's DNS servers).
```